include/inviwo/core/util/colorconversion.h

```diff
@@ -302,7 +302,8 @@ IVW_CORE_API vec3 Luv2XYZ(const vec3 &Luv, vec3 whitePointXYZ = getD65WhitePoint
 /**
  * \brief convert from hcl to rgb.
  *
- * HCL (hue chroma luminance) is converted to RGB (red green blue) values.
+ * HCL (hue chroma luminance) in range [a, b] is converted to RGB (red green blue) in range [a, b],
+ * where usually a = 0 and b = 1.
```

Comment:
Usually :)
I would just write [0 1]. However, is not Hue usually a degree/radian, [-pi pi]?

Reply:
Hue, at least for the HSV color space is typically encoded in degree [0,360). We might, however, use the range [0,1] for the sake of simplicity. Not sure regarding HCL

Reply:
We should specify which range this function expects, not how the color space is "usually" defined ;-)